content/cli/users.md

```diff
@@ -67,7 +66,7 @@ User `sue` created
 
 `cfy users set-password [OPTIONS] USERNAME`
 
-Set the password for a specific user. Use this command in a non-LDAP/AD setup.<br>
+Set the password for a specific user. You can use this command in a non-LDAP setup to change each of the users' passwords, and in LDAP mode to change only the password of the bootstrap-admin (the user created by default upon Manager's installation) .<br>
```

Comment:
You can use this command in a non-LDAP setup to change each of the users' passwords, and in LDAP mode to change only the password of the bootstrap-admin (the user created by default upon Manager's installation) .<br>
->
You can use this command to change user passwords for non-LDAP users, and to change the password of the bootstrap-admin (the user created by default during Manager installation).

Reply:
I prefer separating it to "in LDAP mode" and "in non-LDAP mode", as a saying like "for non-LDAP users" might cause users to think you can have both LDAP and non-LDAP users at the same time.